src/main/java/org/apache/commons/compress/compressors/gzip/GzipParameters.java

```diff
@@ -118,4 +120,22 @@ public int getOperatingSystem() {
     public void setOperatingSystem(final int operatingSystem) {
         this.operatingSystem = operatingSystem;
     }
+
```

Comment:
You're missing a Javadoc comment here.

Reply:
The rest of the attributes in the class only had javadoc for setter method. I had started writing javadoc but removed it to match the rest. I can certainly add.